scanner/src/funTest/kotlin/scanners/ScannerIntegrationFunTest.kt

```diff
@@ -59,18 +59,18 @@ import org.ossreviewtoolkit.utils.test.matchExpectedResult
 import org.ossreviewtoolkit.utils.test.patchActualResult
 
 class ScannerIntegrationFunTest : WordSpec({
-    "scan()" should {
-        val analyzerResult = createAnalyzerResult()
+    "scan() all packages corresponding to a single VCS" should {
```

Comment:
Nit: It should actually say "scanning", but that would not mention the function name anymore. While you probably could write something like "scan()'ing", maybe instead rephrase like

    "For packages with the same provenance, scan()" should

(I'm not sure whether that's the right wording, but you get my point.)

Reply:
I just went for "scanning". I believe it's good enough without mentioning the function name.

Reply:
Ok, but as this is a regular word now that starts the sentence, it should be capitalized (in both tests).